assets/csi_cloud_config.ini

```diff
@@ -0,0 +1,7 @@
+[Global]
```

Comment:
Isn't assets directory intended only for OpenShift object definitions?

Reply:
I don't think so, there is a rule like that.